src/go/types/version/v1/network.go

```diff
@@ -254,6 +259,12 @@ func (this *Interface) SetGateway(gw string) {
 	this.GatewayF = gw
 }
 
+func (this *Interface) SetDNS(dns []string) {
+	for _, value := range dns {
+		this.DNSF = append(this.DNSF, value)
+	}
```

Comment:
@glattercj now that we're using `[]string` instead of `StringSlice`, you can just do `this.DNSF = dns` here. Will save us a for-loop, and keep it consistent with other setters.

Reply:
Rgr